pkg/controlplane/controller/defaultservicecidr/default_servicecidr_controller.go

```diff
@@ -223,9 +237,7 @@ func (c *Controller) syncStatus(serviceCIDR *networkingapiv1.ServiceCIDR) {
 			klog.Infof("error updating default ServiceCIDR status: %v", errApply)
 			c.eventRecorder.Eventf(serviceCIDR, v1.EventTypeWarning, "KubernetesDefaultServiceCIDRError", "The default ServiceCIDR Status can not be set to Ready=True")
 		}
-	} else if !c.reportedMismatchedCIDRs {
-		klog.Infof("inconsistent ServiceCIDR status, global configuration: %v local configuration: %v, configure the flags to match current ServiceCIDR or manually delete the default ServiceCIDR", serviceCIDR.Spec.CIDRs, c.cidrs)
-		c.eventRecorder.Eventf(serviceCIDR, v1.EventTypeWarning, "KubernetesDefaultServiceCIDRInconsistent", "The default ServiceCIDR %v does not match the flag configurations %s", serviceCIDR.Spec.CIDRs, c.cidrs)
-		c.reportedMismatchedCIDRs = true
+	default:
+		// no condition or unknown status
```

Comment:
nit: only unknown status. 

Would have expected this to be flipped to true. But that would change the controller behaviour from what it was before. So this might be out of scope.

Reply:
hmm, this comment is in the wrong place, `// no condition or unknown status` is the previous statement

```
case currentReadyCondition == nil || currentReadyCondition.Status == metav1.ConditionUnknown:
```

Reply:
```suggestion
```

Reply:
let me remove the comment